proto/ptypes/controller.proto

```diff
@@ -59,7 +59,8 @@ message ControllerReplica {
 }
 
 message VolumeStartRequest {
-    repeated string replicaAddresses = 1;
+    int64 size = 1;
+    repeated string replicaAddresses = 2;
```

Comment:
NIT: It's better not to change the sequence number of the existing variables.

Reply:
Exactly. 

For proto, it uses the same design guidance for other native language like C++ or interface definition language. The number is meaningful for backward compatibility.

Reply:
Updated. Thank you.